pages/news/2020-05-16-intro-outro-sections.html

```diff
@@ -0,0 +1,107 @@
+<!-- screenshot -->
+
+It is up to you to decide where to place the intro end and outro start points.
+You can easily jump to the intro end and outro start points by pressing the buttons like hotcues, but that is not their primary purpose.
+Marking the whole intro and outro allows Mixxx to calculate how long each section is and show it on the overview waveform.
+You can use this information to inform when you press play on the next track and so can AutoDJ.
+<!-- screenshot -->
+
+## Intro & Outro Mixing Techniques Before Mixxx 2.3 ##
+
+Before we implemented these intro & outro cues, I calculated how long these sections were in my head.
+After I identified which track I wanted to mix in next, I would find the point in that track where I would want to have the old track completely faded out.
+Then, I would look at what time that was on the deck and keep that number in my memory.
+I would watch the how much time was remaining on the old track and listen for the time to start mixing out the old track.
+
+For example, if I knew the old track had a fairly sudden ending with only 1 or 2 seconds without a lot going on at the end, and the next track had a 20 second intro, to avoid letting the energy of the mix drop if I let only that intro play, I could start the next track when the old track's deck showed 20 seconds remaining.
+This would align the end of the old track with the end of the intro of the new track.
+This way I would know that the energy of the new track would pick up exactly as the old track ends.
+Depending on the musical content of each track, I might or might not bring the fader up on the new track before the last 1 or 2 seconds of the old track.
+
+Before Mixxx 2.3, I could mark the end of the intro with a hotcue, but I still had to jump to that hotcue and look at what time the deck was at to know how long the intro was.
+Although Mixxx could remember the hotcue, I'd still have to go to it every time I loaded the track to remember what position it was at.
+And what if the old track ended with a few seconds of silence or inaudibly quiet sounds?
+My calculation for when to press play on the next track would be thrown off.
+Marking the intro and outro *sections* rather than single points makes both of these much easier.
+
+## Using The New Intro & Outro Cues ##
+
+To do the same mix aligning the ends of the intro & outro sections in Mixxx 2.3, I don't have to keep the length of the intro in my memory anymore; Mixxx shows it to me on the overview waveform.
+Now I can watch the time remaining on the old track and press play when it is equal to the length of the intro of the next track.
+
+If the old track has 1 or 2 seconds of silence at the end, the analyzer will mark the outro end where the sound becomes too quiet to hear.
+In this case, instead of watching the time remaining on the deck, I can hover my mouse cursor over the outro end line on the overview waveform and Mixxx will show me how much time is left until that point.
```

Comment:
```suggestion
add screenshot showing that scenario?
```

Reply:
I suggest to leave the text as-is and add an animated gif (e.g. using [Peek](https://github.com/phw/peek))